[BUG] `bpf-loader-rs` and `ecli` build failed on aarch64-linux

**Describe the bug**

As in the title.
Occurs on both latest release `v0.3.55` and 1976dda760a2dead485a2d5fe29ece9cf0692254.

Also affects `ecli`.

**To Reproduce**
Enter `bpf-loader-rs` directory and build:
```text
user@opulr-aarch64 ~/e/bpf-loader-rs ((v0.3.55))> cargo b
   Compiling bpf-loader-lib v0.1.6 (/home/user/eunomia-bpf/bpf-loader-rs/bpf-loader-lib)
error[E0308]: mismatched types
   --> bpf-loader-lib/src/skeleton/builder.rs:114:41
    |
114 |               open_bpts.btf_custom_path = path_holder
    |  _____________-------------------------___^
    | |             |
    | |             expected due to the type of this binding
115 | |                 .as_ref()
116 | |                 .unwrap()
117 | |                 .as_os_str()
118 | |                 .as_bytes()
119 | |                 .as_ptr() as *const i8;
    | |______________________________________^ expected `*const u8`, found `*const i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

error[E0308]: mismatched types
   --> bpf-loader-lib/src/skeleton/builder.rs:122:41
    |
122 |             open_bpts.btf_custom_path = env_btf.as_bytes().as_ptr() as *const i8;
    |             -------------------------   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `*const u8`, found `*const i8`
    |             |
    |             expected due to the type of this binding
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `bpf-loader-lib` (lib) due to 2 previous errors
```

**Expected behavior**
Successful build.


**Desktop (please complete the following information):**
```
NAME="openEuler"
VERSION="22.03 (LTS-SP1)"
ID="openEuler"
VERSION_ID="22.03"
PRETTY_NAME="openEuler 22.03 (LTS-SP1)"
ANSI_COLOR="0;31"
```

```
cargo --version
cargo 1.70.0 (ec8a8a0ca 2023-04-25)
```

```
uname -r
5.10.0-136.22.0.98.oe2203sp1.aarch64
```

--- a/bpf-loader-rs/bpf-loader-lib/Cargo.toml
+++ b/bpf-loader-rs/bpf-loader-lib/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "bpf-loader-lib"
-version = "0.1.6"
+version = "0.1.7"
 edition = "2021"
 license = "MIT"
 description = "A library to load json-described ebpf programs, and automatically poll outputs from the program"

--- a/bpf-loader-rs/bpf-loader-lib/src/skeleton/builder.rs
+++ b/bpf-loader-rs/bpf-loader-lib/src/skeleton/builder.rs
@@ -116,10 +116,10 @@ impl<'a> BpfSkeletonBuilder<'a> {
                 .unwrap()
                 .as_os_str()
                 .as_bytes()
-                .as_ptr() as *const i8;
+                .as_ptr() as *const _;
         } else if let Some(env_btf) = env_btf_file_path.as_ref() {
             // SAFETY: env_btf_file_path will live until this function returns
-            open_bpts.btf_custom_path = env_btf.as_bytes().as_ptr() as *const i8;
+            open_bpts.btf_custom_path = env_btf.as_bytes().as_ptr() as *const _;
         } else if !vmlinux_btf_exists {
             bail!("All ways tried to find vmlinux BTF, but not found. Please provide the vmlinux btf using env `BTF_FILE_PATH`. (Tried parameter `btf_archive_path`, {}, and {})",BTF_PATH_ENV_NAME,VMLINUX_BTF_PATH);
         };

--- a/documents/src/installation/build.md
+++ b/documents/src/installation/build.md
@@ -65,4 +65,5 @@ $ make ecc
 
 
 - You can check the Makefile at project root for more details: [Makefile](https://github.com/eunomia-bpf/eunomia-bpf/blob/master/compiler/Makefile)
-- You may want to refer to our CI for more build info: [ecc.yml](https://github.com/eunomia-bpf/eunomia-bpf/blob/master/.github/workflows/ecc.yml)
+- You may want to refer to our CI for more build info: [ecc.yml](https://github.com/eunomia-bpf/eunomia-bpf/blob/master/.github/workflows/ecc-binary.yml)
+

--- a/ecli/client/Cargo.toml
+++ b/ecli/client/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "ecli-rs"
-version = "0.2.6"
+version = "0.2.7"
 edition = "2021"
 description = "The client cli wrapper of ecli"
 license = "MIT"

--- a/ecli/ecli-lib/Cargo.toml
+++ b/ecli/ecli-lib/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "ecli-lib"
-version = "0.2.6"
+version = "0.2.7"
 edition = "2021"
 description = "The core implementation of ecli"
 license = "MIT"
@@ -34,7 +34,7 @@ home = "0.5.4"
 thiserror = "1.0.40"
 bpf-oci = "0.1.0"
 # These deps are only needed when `native-client` feature is enabled
-bpf-loader-lib = { path = "../../bpf-loader-rs/bpf-loader-lib", version = "0.1.3", optional = true }
+bpf-loader-lib = { path = "../../bpf-loader-rs/bpf-loader-lib", version = "0.1.7", optional = true }
 wasm-bpf-rs = { version = "0.3.2", optional = true }
 bpf-compatible-rs = { version = "0.1.0", optional = true }
 

--- a/ecli/server/Cargo.toml
+++ b/ecli/server/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "ecli-server"
-version = "0.2.5"
+version = "0.2.7"
 edition = "2021"
 description = "The server cli wrapper of ecli"
 license = "MIT"

